docs/polyline.md

```diff
@@ -9,7 +9,7 @@
 | `strokeColor` | `String` | `#000` | The stroke color to use for the path.
 | `strokeColors` | `Array<String>` | `null` | The stroke colors to use for the path (iOS only). Must be the same length as `coordinates`.
 | `lineCap` | `String` | `round` | The line cap style to apply to the open ends of the path. Possible values are `butt`, `round` or `square`.
-| `lineJoin` | `String` | `round` | The line join style to apply to corners of the path. Possible values are `miter`, `round` or `bevel`.
+| `lineJoin` | `String` | `round` | The line join style to apply to corners of the path. Possible values are `miter`, `round` or `bevel`. Note: lineJoin is not yet supported for GoogleMaps provider on iOS.
```

Comment:
It's `lineJoin` not available or `lineCap` (english is great 🤗)

Reply:
oops!, it's lineCap. :D